packages/safe-core-sdk-utils/package.json

```diff
@@ -31,7 +31,6 @@
   ],
   "homepage": "https://github.com/safe-global/safe-core-sdk#readme",
   "devDependencies": {
-    "@gnosis.pm/safe-deployments": "1.19.0",
```

Comment:
Does it really need to be only a dev dependency?

Reply:
Inside the `safe-core-sdk-utils` package, yes